utils/tox.ini

```diff
@@ -17,3 +17,18 @@ commands =
     unit: pytest {posargs}
     flake8: python setup.py flake8
     pylint: python setup.py lint
+
+[pytest]
```

Comment:
needs a rebase, this went into `setup.cfg` as you suggested (to keep test/lint related config in one file)